napari/utils/notebook_display.py

```diff
@@ -82,13 +82,11 @@ def _clean_alt_text(self, alt_text):
                     'will be stripped altogether without lxml.'
                 )
                 return None
-            alt_text = html.unescape(
-                str(alt_text)
-            )  # cleaner won't recognize unescaped script tags
+            # cleaner won't recognize unescaped script tags
+            alt_text = html.escape(str(alt_text))
             cleaner = Cleaner()
             doc = document_fromstring(alt_text)
```

Comment:
Should `document_fromstring` take a string that represents an HTML element, fragment, or document? E.g. `'<img src="" alt="<>?:"{}|_+"></img>'` instead of just `'<>?:"{}|_+'`)?